server/@types/forms/index.d.ts

```diff
@@ -119,7 +119,7 @@ declare module 'inductionForms' {
   }
 
   export interface WorkInterestRolesForm {
-    workInterestRoles: Map<WorkInterestTypeValue, string>
+    workInterestRoles: [WorkInterestTypeValue, string][]
```

Comment:
Now that we hold the `WorkInterestRolesForm` on the session, we can no longer use a Map as its internal data structure, because a Map is not serializable so it does store/retrieve correctly from the session! 😢 
Changing it to an array of typed array.
(conceptually it's very similar to what we had previously with the Map, but it serializes properly)